README.md

```diff
@@ -15,30 +14,7 @@
    +  Create documents
    +  Delete documents, users
    +  Update documents, users
-
-## Endpoints
```

Comment:
There's a documentation available at root of the url @ http://cjdocs.herokuapp.com